powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/configuration/PowerAuthServiceConfiguration.java

```diff
@@ -141,6 +141,14 @@ public class PowerAuthServiceConfiguration {
     @Max(64)
     private long signatureValidationLookahead;
 
+    /**
+     * When validating the offline (or decimalized) signature, how many digits should a factor-related component have.
+     */
+    @Value("${powerauth.service.crypto.decimalComponentLength}")
+    @Min(1)
+    @Max(64)
```

Comment:
Why is the limit different from this one?

https://github.com/wultra/powerauth-crypto/pull/452/files#diff-5b5c15aab2a6be224c3146ec1141b01c4f1b679eac3b7df30dec2a6767576da2R115